### PR TITLE
Add Quay namespace

### PR DESCRIPTION
**Pull request by dmage**
Create a namespace for Quay images and add users that can manage them.

### CODE REVIEW

File: clusters/app.ci/quay/group-quay-admins.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+kind: Group
+apiVersion: v1
+metadata:
+  name: quay-admins
+users:
+- dmage
```
Comment (dmage) [inline clusters/app.ci/quay/group-quay-admins.yaml:6]: @flavianmissi @ricardomaraschini if you want to have access to this namespace on app.ci, please give me your usernames

Comment (flavianmissi) [reply clusters/app.ci/quay/group-quay-admins.yaml:6]: fmissi

**Comment by dmage** [comment]
/assign @petr-muller
please check that clusters/ resources are ok

**Review by petr-muller — APPROVED** [review]

**Pull request merged**